benchmark/infrastructure/BenchmarkConfigurationShorthandVisitor.cpp

```diff
@@ -0,0 +1,65 @@
+// Copyright 2023, University of Freiburg,
+// Chair of Algorithms and Data Structures.
+// Author: Andre Schlegel (March of 2023, schlegea@informatik.uni-freiburg.de)
+
+#include "../benchmark/infrastructure/BenchmarkConfigurationShorthandVisitor.h"
+
+#include <any>
+
+#include "util/json.h"
+
+std::any ToJsonBenchmarkConfigurationShorthandVisitor::visitShortHandString(
+    BenchmarkConfigurationShorthandParser::ShortHandStringContext* context) {
+  return visitAssignments(context->assignments());
+}
+
+std::any ToJsonBenchmarkConfigurationShorthandVisitor::visitAssignments(
```

Comment:
For example (see what I wrote for the header) this can return a `nlohmann::json`.